README.md

```diff
@@ -29,3 +29,17 @@ Required environment variables:
   HODIS_HOST=https://hodis.datasektionen.se
   LOGIN_HOST=https://login2.datasektionen.se
 ```
+
+## Dependencies
+Depends on login, hodis is refered to but not integral.
+
+## Development
+With docker installed, run `docker-compose up --build`, <localhost.datasektionen.se:5000> will be the local development version.
+Will look for ENV in a file called variables.env.
```

Comment:
`With docker and docker-compose installed ...`
`Will look for environment variables in a file called ...`